Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Get.ts

```diff
@@ -21,17 +21,18 @@ export async function getDocumentByDocumentUuid(
 
   try {
     const result: WithId<MeadowlarkDocument> | null = await mongoCollection.findOne({ documentUuid });
-    if (result === null) return { response: 'GET_FAILURE_NOT_EXISTS', document: {} };
-    // eslint-disable-next-line no-underscore-dangle
-    const documentLastModifiedDate: string | null = result.lastModifiedAt
-      ? new Date(result.lastModifiedAt).toISOString()
-      : null;
+    if (result === null) {
+      return { response: 'GET_FAILURE_NOT_EXISTS', edfiDoc: {}, documentUuid, lastModifiedDate: 0 };
+    }
+
     return {
       response: 'GET_SUCCESS',
-      document: { id: documentUuid, ...result.edfiDoc, _lastModifiedDate: documentLastModifiedDate },
+      edfiDoc: result.edfiDoc,
+      documentUuid,
+      lastModifiedDate: result.lastModifiedAt,
```

Comment:
Changed GetResult in meadowlark-core to require id and lastModifiedDate explicitly.